queries/highlights.scm

```diff
@@ -131,21 +158,24 @@
   (true)
 ] @boolean
 
-;; Tables
+; Tables
+(field
+  name: (identifier) @property)
 
-(field name: (identifier) @field)
-
-(dot_index_expression field: (identifier) @field)
+(dot_index_expression
+  field: (identifier) @variable.member)
```

Comment:
I don't have any inputs on these ones... since I'm not very up to date on the latest conventions for these. If these conform to the latest conventions, looks alright 👍🏼